pkg/lib/oauth/resolver.go

```diff
@@ -243,6 +243,7 @@ func (re *Resolver) accessOfflineGrant(offlineGrant *OfflineGrant, accessEvent a
 	if err != nil {
 		return nil, err
 	}
+	offlineGrant.ExpireAtForResolvedSession = expiry
```

Comment:
Looks like it could be easy for us to forget updating `ExpireAtForResolvedSession`.
Should we update `ExpireAtForResolvedSession` in `ComputeOfflineGrantExpiry`?

For idp session maybe we can do it in `computeSessionStorageExpiry`?